osu.Game.Rulesets.Taiko.Tests/osu.Game.Rulesets.Taiko.Tests.csproj

```diff
@@ -11,5 +11,6 @@
   </PropertyGroup>
   <ItemGroup Label="Project References">
     <ProjectReference Include="..\osu.Game.Rulesets.Taiko\osu.Game.Rulesets.Taiko.csproj" />
+    <ProjectReference Include="..\osu.Game.Tests\osu.Game.Tests.csproj" />
```

Comment:
Curious how this doesn't result in a cyclic reference, because `osu.Game.Tests` references this project.

Reply:
It does? I believe it references `osu.Game.Rulesets.Taiko`, not `osu.Game.Rulesets.Taiko.Tests`...?

Reply:
Oh, indeed, whoops :)